region/lowernorfair/west.json

```diff
@@ -110,19 +153,50 @@
           "to": [
             {
               "id": 1,
-              "requires": null,
+              "requires": [ 
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "UsedAcidChozoStatue",
+                {"cost": [
+                  {"heatFrames": 300}
+                ]}                
+              ],
               "unlock": null
             },
             {
               "id": 3,
-              "requires": null,
+              "requires": [ 
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "UsedAcidChozoStatue",
+                {"cost": [
+                  {"heatFrames": 250}
+                ]}                
+              ],
               "unlock": null
             },
             {
               "id": 5,
-              "requires": [
+              "requires": [ 
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
                 "UsedAcidChozoStatue",
-                "canPassBombPassages"
+                "canPassBombPassages",
+                {"cost": [
+                  {"heatFrames": 250},
+                  {"ammo": [
```

Comment:
No passage with just Morph Bombs?

Reply:
Looks like I forgot that one yes. To the bat-emulator-mobile!